cypress/integration/page.spec.js

```diff
@@ -1,7 +1,81 @@
-// WRITE TESTS HERE
+const goodsFromServer = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+const buttons = {
+  clear: 'Clear',
+  remove: 'Remove',
+  select: 'Select'
+};
+
+Cypress.Commands.add('selectItemBtn', (btn) => {
+  cy.contains('li', goodsFromServer[8])
+    .contains(btn);
+});
```

Comment:
Не варто робити команди для 1 тесту, ще й із захардкодженим індексом
Досить буде зробити
```suggestion
const page = {
  getGoodButton(index, buttonSelector) {
    return cy.contains('li', goodsFromServer[index])
      .contains(buttonSelector);
    });
  },
};
```

Reply:
дан